tooling/scripts/tests/generate_matrix_test.sh

```
@@ -13,17 +13,27 @@ function assert_equals {
   fi
 }
 
+function beforeAll {
+  export FILE_EXISTS_CMD=$SCRIPT_DIR/mock_cmd.sh
+  export CAT_CMD=$SCRIPT_DIR/mock_cmd.sh
+}
+
+function beforeEach {
+  export MOCK_ARGUMENT_FILE=$(mktemp)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)